code-studio/src/js/components/progress/progress_dot.jsx

```diff
@@ -147,12 +147,14 @@ export const ProgressDot = React.createClass({
 
   render() {
     const level = this.props.level;
-    const uid = level.uid || level.id.toString();
+    const onCurrent = this.props.currentLevelId &&
+        ((level.ids && level.ids.map(id => id.toString()).indexOf(this.props.currentLevelId) !== -1) ||
+        level.uid === this.props.currentLevelId);
```

Comment:
We've ended up in a weird place where we have both `ids` and `uid`.  Should `ids` really be `uids`?

If one variant of a level is a `LevelGroup` what will the `ids` and `uid` be for that dot?